samples/sid_end_device/include/cli/app_shell_events.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef APP_DUT_H
+#define APP_DUT_H
+
+#include <sidewalk.h>
+
+void dut_sidewalk_event_init_handler(void *ctx, void *state_machine);
```

Comment:
minor: make names shorter

Reply:
Renamed the handlers now they follow the schema `state` _ event _ `Event name`